packages/components/psammead-section-label/src/index.jsx

```diff
@@ -133,6 +135,7 @@ SectionLabel.propTypes = {
   service: string.isRequired,
   visuallyHidden: bool,
   backgroundColor: string,
+  overrideHeadingAs: string,
```

Comment:
If that is possible, definately

Reply:
@andrewscfc would a prop validation warning suffice? or should it be enforced?

Reply:
I'd just say prop validation personally, just to guide people in the right direction. Enforcing is a bit pointless because anyone can edit the code, it's just to encourage people to pause and think really.